.changeset/gold-lions-clean.md

```diff
@@ -0,0 +1,6 @@
+---
+"@liam-hq/db-structure": patch
+"@liam-hq/cli": patch
```

Comment:
```suggestion
```

This line doesn't need because it is now set up automatically.

- ref: https://github.com/liam-hq/liam/pull/673